### ISSUE TITLE
CSS button style consideration

### ISSUE THREAD

**Issue by PabloCorso** [Kiarash-Z/react-modern-calendar-datepicker#128]
There is a style under `src/DatePicker.css` that overrides all button styles. Would it be ok to have a more specific selector for it?

For example, it will target all buttons inside the renderFooter of the Calendar. It makes it much more difficult to override the footer button background since it is targeting button elements directly. I can't just import and use some external styles for the button, I got to override it manually instead.

```css
/* DatePicker.css line 92 */
.Calendar button {
  font-family: inherit;
  background: transparent;
  cursor: pointer;
  -webkit-tap-highlight-color: transparent;
  outline: none;
}
```

**Comment by Kiarash-Z** [comment]
@PabloCorso You're absolutely right. I will fix this in the next version. Thanks for reporting. I'll keep this issue open till the fix release.

**Comment by Kiarash-Z** [comment]
Fixed on the latest version(3.0.3) 🎉
